tests/flow/test_params.py

```diff
@@ -0,0 +1,53 @@
+import os
+import sys
+from redisgraph import Graph, Node
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from base import FlowTestsBase
+sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/../../')
+from demo import QueryInfo
+
+GRAPH_ID = "G"
+redis_graph = None
+
+class testParams(FlowTestsBase):
+    def __init__(self):
+        super(testParams, self).__init__()
+        global redis_graph
+        redis_con = self.env.getConnection()
+        redis_graph = Graph(GRAPH_ID, redis_con)
+
+    def setUp(self):
+        self.env.flush() 
+    
+    def test_simple_params(self):
+        params = [1, 2.3, "str", True, False, None, [0, 1, 2]]
+
+        for param in params:
+            query = "RETURN $param"
```

Comment:
`query` can be defined outside of this loop.

Reply:
fixed

Reply:
Ah, strange! So be it, though.